terraform/deployments/govuk-publishing-platform/backends_origin.tf

```diff
@@ -0,0 +1,177 @@
+module "backends_origin" {
+  source = "../../modules/origin"
+
+  providers                            = { aws = aws, aws.us_east_1 = aws.us_east_1, random = random }
+  name                                 = "backends"
+  vpc_id                               = local.vpc_id
+  aws_region                           = data.aws_region.current.name
+  assume_role_arn                      = var.assume_role_arn
+  public_subnets                       = local.public_subnets
+  public_zone_id                       = aws_route53_zone.workspace_public.zone_id
+  external_app_domain                  = aws_route53_zone.workspace_public.name
+  subdomain                            = "backends"
+  extra_aliases                        = compact([local.is_default_workspace ? "publisher.${var.publishing_service_domain}" : null, "publisher.${aws_route53_zone.workspace_public.name}", "signon.${aws_route53_zone.workspace_public.name}"])
+  load_balancer_certificate_arn        = aws_acm_certificate_validation.workspace_public.certificate_arn
+  cloudfront_certificate_arn           = aws_acm_certificate_validation.public_north_virginia.certificate_arn
+  publishing_service_domain            = var.publishing_service_domain
+  workspace                            = local.workspace
+  is_default_workspace                 = local.is_default_workspace
+  allowlist_cidrs                      = local.is_default_workspace ? concat(var.office_cidrs_list, data.fastly_ip_ranges.fastly.cidr_blocks, local.aws_nat_gateways_cidrs) : concat(var.office_cidrs_list, local.aws_nat_gateways_cidrs)
+  rails_assets_s3_regional_domain_name = aws_s3_bucket.backends_rails_assets.bucket_regional_domain_name
+
+  fronted_apps = {
+    "publisher" = { security_group_id = module.publisher_web.security_group_id },
+    "signon"    = { security_group_id = module.signon.security_group_id },
+  }
+}
+
+## Publisher
+
+resource "aws_lb_target_group" "publisher" {
+  name        = "backends-origin-publisher-${local.workspace}"
+  port        = 80
+  protocol    = "HTTP"
+  vpc_id      = local.vpc_id
+  target_type = "ip"
+
+  health_check {
+    path = "/healthcheck"
+  }
+}
+
+resource "aws_lb_listener_rule" "publisher" {
+  listener_arn = module.backends_origin.origin_alb_listerner_arn
+  priority     = 1
```

Comment:
Can we reserve rule priorities for future use? Otherwise we might need to do a hairy state modification later?

Priority 1 seems like it would be quite useful.

Reply:
cool, I can start at `11`